tests/storage/blocksd_test.py

```diff
@@ -1079,7 +1079,148 @@ def test_create_with_bitmaps(
 
 @requires_root
 @pytest.mark.root
-def test_failed_to_add_bitmaps_to_v3_domain(
+@pytest.mark.parametrize("domain_version", [4, 5])
+def test_create_snapshot_with_new_bitmap(
+        tmp_storage, tmp_repo, fake_access, fake_rescan, tmp_db, fake_task,
+        fake_sanlock, domain_version):
+    sd_id = str(uuid.uuid4())
+
+    dev = tmp_storage.create_device(20 * GiB)
+    lvm.createVG(sd_id, [dev], blockSD.STORAGE_UNREADY_DOMAIN_TAG, 128)
+    vg = lvm.getVG(sd_id)
+
+    dom = blockSD.BlockStorageDomain.create(
+        sdUUID=sd_id,
+        domainName="domain",
+        domClass=sd.DATA_DOMAIN,
+        vgUUID=vg.uuid,
+        version=domain_version,
+        storageType=sd.ISCSI_DOMAIN)
+
+    sdCache.knownSDs[sd_id] = blockSD.findDomain
+    sdCache.manuallyAddDomain(dom)
+
+    dom.refresh()
+    dom.attach(tmp_repo.pool_id)
+
+    img_id = str(uuid.uuid4())
+    base_id = str(uuid.uuid4())
+    base_capacity = GiB
```

Comment:
Arguably better (here & lines 1196, 1309):
`1 * GiB`

Reply:
If we had several size and using 1 * GiB helps to align the values, yes.